### DISCUSSION TITLE
Info about DISPLAY ASPECT RATIO for playing movie

### DISCUSSION THREAD

**Discussion by olvrteil** [wizmo2/zidoo-player#29] [Ideas]
I, first thank you for this integration. I will use it to manage my room lighting when movie play, pause, stop.

I have an idea to automate the lens shift & zoom of a videoprojector from the playing movie format.
Like the zidoo interface is designed to work in 16/9, if you want to have a correct 16/9 image on a 2:35 screen, you must use the lens shit and zoom to adapt image on frame.
Then, you launch a movie and movie is in 2:35 or maybe 1:85, again you must change lens and zoom to adapt image on screen.

Would it be possible to have in Home assistant the format of currently playing movie ?
It would allow to change the videoprojector memory preset to match what's on screen. Then back to the menu, back to 16/9 videoprojector memory preset

![image](https://github.com/user-attachments/assets/37ede96c-cccc-4c13-b73b-3c740153f827)
![image](https://github.com/user-attachments/assets/4f3765c4-e7cc-4177-b02c-8dc2f247f627)

**Comment by wizmo2** [comment]
When playing a video, it should report the width and height of the video format, which can be used to calcukate the aspect ratio.

Have a look at [the notes](https://github.com/wizmo2/zidoo-player/blob/main/notes.md#using-additional-attributes-dev) for an example of how this can be used in an automation.
